### PR TITLE
Being compatible with Windows

### PR DESCRIPTION
**Pull request by znsoftm**
A few minor adjustments are needed to ensure compatibility with Windows and Visual Studio 2022.

### CODE REVIEW

**Comment by skeskinen** [comment]
Hi! Thanks for the contribution, looks interesting. Just a couple things to check to ensure it still builds on other systems.

File: examples/test_tokenizer.cpp [hunk 1]
```diff
@@ -27,7 +28,7 @@ void tokenizer_test(bert_ctx * ctx, const std::string& input, const std::vector<
         printf("]\n");
 
         for (size_t i = 0; i < result.size(); i++) {
-            bert_vocab_id a = expected[std::min(i, expected.size()-1)];
+            bert_vocab_id a = expected[min(i, expected.size()-1)];
```
Comment (skeskinen) [inline examples/test_tokenizer.cpp:31]: Why is std:: removed here? This doesn't compile on linux

Comment (znsoftm) [reply examples/test_tokenizer.cpp:31]: Can you test it ? I'll test it again for linux.

Comment (znsoftm) [reply examples/test_tokenizer.cpp]: fixed. Need a definition to avoid windows using another macro.

**Pull request merged**